### PR TITLE
Upgrade

### PR DESCRIPTION
**Pull request by plicjo**

### CODE REVIEW

**Comment by plicjo** [comment]
my bad

File: LICENSE.md [hunk 1]
```diff
@@ -1,7 +1,5 @@
 MIT License
 
-Copyright (c) 2017 Hashrocket
```
Comment (tmock12) [inline LICENSE.md:3]: @plicjo tsk tsk tsk, you can't modify the license, only sublicense

tsk tsk tskt

Comment (plicjo) [reply LICENSE.md:3]: Got you 💪

Comment (plicjo) [reply LICENSE.md:3]: I just added it back

**Pull request closed without merging**